var/spack/repos/builtin/packages/sim-model/package.py

```diff
@@ -139,23 +156,34 @@ def _install_src(self, prefix):
         """Copy original and translated c mods
         """
         arch = os.path.basename(self.neuron_archdir)
+        mkdirp(prefix.lib.mod, prefix.lib.hoc, prefix.lib.python)
         copy_all('mod', prefix.lib.mod)
         copy_all('hoc', prefix.lib.hoc)
+        if os.path.isdir('python'):  # Recent neurodamus
+            copy_all('python', prefix.lib.python)
 
-        for cmod in find(arch, "*.c", recursive=False):
+        for cmod in find(arch, '*.c', recursive=False):
             shutil.move(cmod, prefix.share.modc)
 
-        if self.spec.satisfies('+coreneuron'):
-            shutil.move(join_path(arch, 'core/mod2c'), prefix.share.modc_core)
+    def _setup_environment_common(self, spack_env, run_env):
+        spack_env.unset('LC_ALL')
+        # Remove LD_LIB_PATHs
+        to_rem = ('LD_LIBRARY_PATH', 'DYLD_LIBRARY_PATH', 'DYLD_FALLBACK_LIBRARY_PATH')
```

Comment:
`to_rem` is ambiguous. I read it as `to_remember`. Please change to `to_rm` or other.

Reply:
dont like my var names? 😄  (even though to_rm sounds better, code is not too complex to understand)

Reply:
new commit changes to to_rm